source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -94,13 +94,7 @@ It is used at the data acquisition and data preparation stages to:
   device's local memory. Therefore, a data source enables complex data analytics
   scenarios, such as :capterm:`online computations <Online mode>`.
 
-- Filter and normalize :capterm:`feature` values that are being extracted.
-
-- Recover missing :capterm:`feature` values.
-
-- Detect :capterm:`outliers <outlier>` and recover the abnormal data.
-
-- Transform datasets into numerical representation. Data source shall
+- To transform datasets into numerical representation. Data source shall
```

Comment:
why '.' after representation?

Reply:
It is actually two separate sentences, overall it looks like this:

It is used at the data acquisition and data preparation stages for the
following:

- To extract datasets from external sources such as databases, files, remote
  storage.

- To load datasets into the device's local memory. Data do not always fit
  the local memory, especially when processing with accelerators. A data source
  provides the ability to load data by batches and extracts it directly into the
  device's local memory. Therefore, a data source enables complex data analytics
  scenarios, such as :capterm:`online computations <Online mode>`.

- To transform datasets into numerical representation. Data source shall
  automatically transform non-numeric :capterm:`categorical <categorical
  feature>` and :capterm:`continuous <continuous feature>` data values into one
  of the numeric :capterm:`data formats <data format>`.

Reply:
It this two different sentences.